provisioners/configure-backend.sh

```diff
@@ -0,0 +1,4 @@
+./configure.sh
+echo "Configure Notification Service"
+envsubst < $TEMPLATES_PATH/etc/systemd/system/notification.service > /etc/systemd/system/notification.service
+systemctl enable notifications.service
```

Comment:
> `Failed to enable unit: Unit file notifications.service does not exist.`

We need to use the same (lack of) plurality throughout:

```suggestion
systemctl enable notification.service
```